github.js

```diff
@@ -163,11 +186,14 @@ async function ensurePrsExist(sg, allBranches, combinedBranch, remote = DEFAULT_
         title,
         body,
       };
-      process.stdout.write(`Creating PR for branch "${branch}"...`);
+      const baseMessage = base === baseBranch ? colors.dim(` (against ${base})`) : '';
```

Comment:
Having the base branch in the console output is helpful for visually confirming what base branch is being used since it can now change.